app/Temperature.hs

```diff
@@ -0,0 +1,11 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+import           Data.Text (unpack)
+import           Turtle
+
+main :: IO ()
+main = sh $ liftIO . putStrLn =<< cpuTemperature
+
+cpuTemperature :: Shell String
+cpuTemperature = unpack <$> (strict $
+  inshell ("sensors | grep -oP 'Package.*?\\+\\K[0-9.°C]+'") mempty)
```

Comment:
in case that you are interested, you can grep with turtle as well (we use that in the memory module)